include/roboteam_ai/stp/Skill.h

```diff
@@ -56,20 +56,20 @@ namespace rtt::ai::stp {
          * Terminates the skill
          * @return Status of termination
          */
-        virtual Status onTerminate() noexcept;
+        virtual Status onTerminate() noexcept {};
```

Comment:
I think this should be purely virtual
```suggestion
        virtual Status onTerminate() noexcept = 0;
```

Reply:
I was thinking about this. It is cleaner to make it pure, so it is the responsibility of the derived skills. But the skills mostly do not need termination/initialization, so having it not pure is just convenience. If you think making it pure would be better, I'll go with that.